spec/controllers/media_controller_spec.rb

```diff
@@ -79,4 +85,43 @@
       end
     end
   end
+
+  describe '#verify_token' do
+    let(:id) { 'ab123cd4567' }
+    let(:file_name) { 'interesting_video.mp4' }
+    let(:user_ip_addr) { '192.168.1.100' }
+    let(:token) { StacksMediaToken.new(id, file_name, user_ip_addr) }
+
+    # rubocop:disable Metrics/LineLength
```

Comment:
The departmental best practice is to use the `.rubocop_todo.yml` file (possibly using rubocop's `--auto-gen-config`).

I would be interested if there would be a way to write these tests (and related code) in a way that fell within the line-length limitation (with 120 being pretty generous).

@jmartin-sul, more than happy to chat or pair on ways to do that while keeping the code readable if you would like.